core/templates/dev/head/pages/exploration_editor/editor_tab/add_answer_group_modal_directive.html

```diff
@@ -43,6 +43,11 @@ <h3>Add Response</h3>
                                 add-state="addState">
     </outcome-destination-editor>
   </form>
+
+  <div class="oppia-correctness-label-editor" ng-if="isCorrectnessFeedbackEnabled() && !isCurrentInteractionLinear()" style="margin-left: 4px; margin-top: 18px;">
+    <strong>The answers in this group are correct</strong>
+    <input class="protractor-test-editor-correctness-toggle" type="checkbox" ng-model="tmpOutcome.labelledAsCorrect">
```

Comment:
is this classed used anywhere? `protractor-test-editor-correctness-toggle`

Reply:
Yes, it is used in protractor tests - ExplorationEditorMainTab.js.